### ISSUE TITLE
Hoyebridge-Hue can't connect to deConz gateway

### ISSUE THREAD

**Issue by tomsteenbakkers** [ebaauw/homebridge-hue#343]
I have received my new Raspbee this week. I have installed it and after install I started the deConz GUI. and it's connected directly to the Raspbee.

With the Phoscon app I paired a sensor (Xiaomi door sensor) successfully. I can also see in the deConz GUI de new sensor.

Then I rebooted the RPi and  I get the following error message for `homebridge-hue`. Looking at the log of `homebridge` I see the log messages from P1 but `homebridge` services is not started. It is also not possible to start the deConz GUI, after a restart it starts again but it can't connect to the Raspbee anymore. I can, however, start the Phoscon web app and when I change the state of the sensor the Phoscon app is responding well.

Any ideas?

#### Log Messages

> [2018-5-16 21:20:54] [Hue] dresden appspot ipv6: communication error ENETUNREACH on https://dresden-light.appspot.com/
> [2018-5-16 21:20:59] [Hue] RasberryPi: gateway request 2: post / {"devicetype":"homebridge-hue#RaspberryPi"}
> [2018-5-16 21:20:59] [Hue] RasberryPi: gateway http status 403 Forbidden

**Comment by ebaauw** [comment]
Homebridge-hue tries to create a username (API key) to connect to deCONZ, but you haven’t unlocked the gateway.

**Comment by tomsteenbakkers** [comment]
@ebaauw All problems solved, deConz and Hue are now working together. I only get a IPv6 error/warning during startup. Tomorrow I will add some sensors. 

Thanks for the support 👋

**Comment by tomsteenbakkers** [comment]
Still one problem: I have added succesfully 2 Xiaomi sensors to deConz. During startup the homebridge-hue plugin recognise the 2 accessory, but they no appear in the home app and eve app. The two devices where paired with the Xiaomi gateway before I added them to deConz. I do not see them in the Mi app anymore. Is there a way to force homebridge the forget old devices or something like that?
[homebridge-hue.json.txt](https://github.com/ebaauw/homebridge-hue/files/2016587/homebridge-hue.json.txt)
